src/__tests__/shallow.test.js

```diff
@@ -1,7 +1,8 @@
 // @flow
 import React from 'react';
 import { View, Text } from 'react-native';
-import { shallow, render } from '..';
+import { render } from '..';
+import shallow from '../shallow';
```

Comment:
do we have a test for shallow debug? If so, then we could remove it from here

Reply:
Yes, we have. I'll remove this test.